spec/controllers/admin/enterprises_controller_spec.rb

```diff
@@ -144,6 +144,73 @@ module Admin
           expect(distributor.users).to_not include user
         end
 
+        describe "#stripe_connect" do
+          before do
+            allow(controller).to receive(:spree_current_user) { distributor_manager }
+            allow(::Stripe::OAuth).to receive(:authorize_url) { "some_url" }
+          end
+
+          it "redirects to Stripe Authorization url constructed OAuth" do
+            spree_get :stripe_connect
+            expect(response).to redirect_to "some_url"
+          end
+        end
+
+        context "#stripe_connect_callback" do
+          let(:params) { { id: distributor.permalink } }
+          before { allow(controller).to receive(:spree_current_user) { distributor_manager } }
+
+          context "when the connector raises a StripeError" do
+            before do
+              allow(Stripe::AccountConnector).to receive(:new).and_raise Stripe::StripeError, "some error"
+            end
+
+            it "returns a 500 error" do
+              spree_get :stripe_connect_callback, params
+              response.status.should be 500
```

Comment:
please, we should use the current `expect(...).to eq()` syntax :pray:

Reply:
Not in `spec/controllers/admin/stripe_accounts_controller_spec.rb`

Resolved